### PR TITLE
test: test to make a min reproducible example

### PR DESCRIPTION
**Pull request by anxolin**
Attempt to reproduce the issue reported by @alfetopito in his PR: 

What I did is to create `test.ts` as a smaller example that reproduces the issue.

What i verified.
- I reproduce it. It happens if I use `getTokenPermitInfo` from `alfetopito-permit-utils' library
- If i don't use it, and I use ethersJS it works

image

Test it by running:
`INFURA_API_KEY=<KEY> npx ts-node-esm src/permitInfo/test.ts`

## Next steps
I assume the issue is with `alfetopito-permit-utils' library. @alfetopito 

a good test is to see how to setup the `globalObject` (this is webpack, but i assume that vite is doing sth similar)

 https://stackoverflow.com/questions/64639839/typescript-webpack-library-generates-referenceerror-self-is-not-defined

### CODE REVIEW

**Comment by anxolin** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by anxolin** [comment]
recheck

File: tsconfig.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "compilerOptions": {
-    "module": "ESNext",
-    "moduleResolution": "nodenext"
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext"
```
Comment (alfetopito) [inline tsconfig.json:4]: Is this needed?
Should I change that too?

Comment (alfetopito) [reply tsconfig.json:4]: I see it in the other PR now

**Pull request closed without merging**